.github/workflows/ci.yml

```diff
@@ -59,6 +60,12 @@ jobs:
         with:
           github-token: ${{ secrets.ELEVATED_GITHUB_TOKEN }}
           no-restore: true # don't download them on a cache hit
+      - if: ${{ !contains(github.event.pull_request.labels.*.name, 'checkout-head') }}
```

Comment:
Could you please add either `name:` fields or comments explaining why we're doing this? Just to save someone figuring it out months from now :D